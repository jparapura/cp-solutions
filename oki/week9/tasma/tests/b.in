3
3
7 7 7
3
7 3 7
5
2 2 3 2 2
