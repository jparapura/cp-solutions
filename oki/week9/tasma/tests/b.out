BRAK
1
2
